config/application.yml.default

```diff
@@ -277,6 +277,7 @@ development:
   password_pepper: f22d4b2cafac9066fe2f4416f5b7a32c
   piv_cac_verify_token_secret: ee7f20f44cdc2ba0c6830f70470d1d1d059e1279cdb58134db92b35947b1528ef5525ece5910cf4f2321ab989a618feea12ef95711dbc62b9601e8520a34ee12
   rails_mailer_previews_enabled: 'true'
+  rack_timeout_service_timeout_seconds: 9_999_999_999
```

Comment:
do we need to add this to `test` as well?

Reply:
I figure the problem happened most often in development? But yeah I guess it wouldn't hurt to add to test... will follow up

Reply:
https://github.com/18F/identity-idp/pull/5766